packages/utils/node-resolver-core/test/resolver.js

```diff
@@ -850,6 +893,16 @@ describe('resolver', function () {
               fileName: 'node_modules/side-effects-false',
               aboveFilePath: path.join(rootDir, 'foo.js'),
             },
+            {
+              fileName: 'package.json',
+              aboveFilePath: path.join(
+                rootDir,
+                'node_modules',
+                'side-effects-false',
+                'src',
+                'index.js',
+              ),
+            },
```

Comment:
@devongovett Does `aboveFilePath` also stop the watching in parent folder after encountering `node_modules` (like `loadConfig` does)? Or is this fine?